lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2078,7 +2119,8 @@ LogicalResult StmtEmitter::visitSV(VerbatimOp op) {
 
     // Emit each chunk of the line.
     emitTextWithSubstitutions(
-        lhsRhs.first, op, [&](Value operand) { emitExpression(operand, ops); });
+        lhsRhs.first, op, [&](Value operand) { emitExpression(operand, ops); },
+        symOps, &names);
```

Comment:
Doesn't the expr form need a similar change in visitVerbatimExprOp?  If you do this, is there any reason to make the name table optional?  Also, I think that would further encourage you to do the symbol resolution inside of emitTextWithSubstitutions